paymentsheet/src/main/java/com/stripe/android/paymentsheet/PaymentMethodsUI.kt

```diff
@@ -222,11 +222,11 @@ private fun PaymentMethodIconUi(
     iconRes: Int,
     iconUrl: String?,
     imageLoader: StripeImageLoader,
-    tintOnSelected: Boolean,
+    iconRequiresTinting: Boolean,
     color: Color,
 ) {
-    val colorFilter = remember(tintOnSelected, color) {
-        if (tintOnSelected) {
+    val colorFilter = remember(iconRequiresTinting, color) {
+        if (iconRequiresTinting) {
             ColorFilter.tint(color)
         } else {
             null
```

Comment:
this code snippet makes so much more sense to me now! When the icon requires tinting, there's a color filter. When it doesn't, there's no color filter. Niceeee. Thanks for doing this!